### PR TITLE
[[ Docs ]] Tweak error reporting and write docs parser warnings to stdout

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: ide-support/revdocsparser.livecodescript [hunk 1]
```diff
@@ -565,19 +577,29 @@ function revDocsGenerateDocsFileFromModular pText
...
+   end if
+   
    repeat for each key tKey in tAPIData
-      put revDocsFormatInlineComments(tAPIData[tKey], false) & CR & CR after tOutput
+      local tFormatted
+      put revDocsFormatInlineComments(tAPIData[tKey], false) into tFormatted
+      if tFormatted is empty then
+         logWarning "No docs for element" && tKey, pFilename
+      else
+         put tFormatted & CR & CR after tOutput
```
Comment (peter-b) [inline ide-support/revdocsparser.livecodescript]: Please use `return` instead of `CR` here.

**Comment by peter-b** [comment]
@livecode-vulcan review ok bdb645b

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok bdb645b8c727af8cf62ba4901d66abadaf199511

**Comment by livecode-vulcan** [comment]
😎 test success bdb645b8c727af8cf62ba4901d66abadaf199511
- try-linux-x86: [success](https://vulcan.livecode.com/builders/try-linux-x86/builds/751)
- try-win-x86: [success](https://vulcan.livecode.com/builders/try-win-x86/builds/811)
- try-ios: [success](https://vulcan.livecode.com/builders/try-ios/builds/701)
- try-android-armv6: [success](https://vulcan.livecode.com/builders/try-android-armv6/builds/765)
- try-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-linux-x86_64/builds/696)
- try-mac: [success](https://vulcan.livecode.com/builders/try-mac/builds/688)

**Pull request merged**